fiat/customprices.go

```diff
@@ -0,0 +1,18 @@
+package fiat
+
+import (
+	"context"
+	"time"
+)
+
+// customPrices implements the fiatBackend interface.
+type customPrices struct {
+	entries []*Price
+}
+
+// rawPriceData returns the custom price point entries.
+func (c *customPrices) rawPriceData(_ context.Context, _,
+	_ time.Time) ([]*Price, error) {
+
+	return c.entries, nil
+}
```

Comment:
Could filter the data given the times here but seems unnecessary? If anything, maybe they should be filtered in frcli so that only the necessary data is sent across?

Reply:
Yeah that sounds like a good idea!